src/enhetsportefolje/enhet-kolonner.tsx

```diff
@@ -132,6 +133,16 @@ function EnhetKolonner({className, bruker, enhetId, filtervalg, valgteKolonner,
                 skalVises={valgteKolonner.includes(Kolonne.OPPFOLGINGSTARTET)}
                 dato={oppfolgingStartetDato(bruker.oppfolgingStartdato)}
             />
+            <TekstKolonne
+                className="col col-xs-2"
+                skalVises={valgteKolonner.includes(Kolonne.CV_KAN_DELES_STATUS)}
+                tekst={cvKanDelesStatusJa ? 'JA' : '-'}
+            />
+            <TekstKolonne
+                className="col col-xs-2"
+                skalVises={valgteKolonner.includes(Kolonne.CV_SVARFRIST)}
+                tekst={filtervalg.stillingFraNavFilter?.at(1)}
+            />
```

Comment:
Som sagt tidligere så er den første kolonnen litt teit, fordi den vil ALLTID vise JA. Fordi hvis den ikke er ja vil vi ikke ha noe data i feltene og da vil ikke kolonnen dukke opp... Hvis du vil ha en grundig forklaring av det kan du se på den siste kommentaren min her: https://trello.com/c/glkNCurk/231-filter-for-svarstatus-for-stilling-fra-nav

Reply:
Hadde det vert mulig å droppe kolonna heilt siden filteret uansett fjerner brukere som ikkje har delt CV med Nav når det blir aktivert? Eller var det kanskje ønskelig at denne informasjonen også skulle komme fram i ei kolonne?

Reply:
Ja, helt enig. Vi diskuterte det på OBOkaffi også 👍